packages/types/src/extrinsic/types.ts

```diff
@@ -105,5 +105,7 @@ declare module '@polkadot/types/interfaceRegistry' {
     FeeExchange: FeeExchange;
     'Option<FeeExchange>': Option<FeeExchange>;
     'Vec<(AssetId, AssetInfo)>': Vec<ITuple<[AssetId, AssetInfo]>>;
+    '(Balance, Balance)': '(Balance, Balance)';
```

Comment:
This continues to confuse me 🤔

Reply:
Managed to fix this.. with any.. have removed it from types :)